public/data/Pritam1812.json

```
@@ -0,0 +1,75 @@
+{
+    "name": "Pritam Hore",
+    "type": "personal",
+    "bio": "Aspiring Software Developer|Open Source Enthusiast!",

```

I think you should add spaces between the `|` 😅 What do you think?
```suggestion
    "bio": "Aspiring Software Developer | Open Source Enthusiast!",
```

---

Sorry for Making Some Silly Mistakes!